### PR TITLE
Add option to Suspend Project

### LINKED ISSUE #377: Allow user to suspend project so it doesnt consume resources
### Epic

#787

### Description

**As a:** Owner

**I want to:** have the ability to stop a container for a project

**So that:** I am not charged for consuming resources 

### Which customers would this be availble to

- [X] All Users, (CE)
- [ ] Specific Project Types (EE)
- [ ] Specific Team Tiers (EE)
- [ ] Other (See comments)

### Acceptance Criteria

- [ ] Container is stopped/removed
- [ ] Project is removed from billing

### PR DESCRIPTION
**Pull request by hardillb**
This adds an option under the Project Settings Danger page. It asks for the same confirmation as delete

fixes #377

### CODE REVIEW

**Comment by hardillb** [comment]
fixes #837

File: frontend/src/pages/project/Settings/Danger.vue [hunk 1]
```diff
@@ -78,6 +78,19 @@
             </div>
         </div>
 
+        <FormHeading class="text-red-700">Suspend Project</FormHeading>
+        <div class="flex flex-col lg:flex-row max-w-2xl space-y-4">
+            <div class="flex-grow">
+                <div class="max-w-sm pt-2">
+                    Once suspended, your project will not be available until restarted.
```
Comment (joepavitt) [inline frontend/src/pages/project/Settings/Danger.vue:85]: Would like to see some extra clarity over what "Suspend" does here. Think we discussed mentioning of billing here too.

**Review by joepavitt — CHANGES_REQUESTED** [review]

File: frontend/src/pages/project/Settings/Danger.vue (hunk 1 above)
Comment (hardillb) [reply frontend/src/pages/project/Settings/Danger.vue:85]: We did, but I would like to gate that on having billing enabled

File: frontend/src/pages/project/Settings/Danger.vue [hunk 2]
```diff
@@ -78,6 +78,19 @@
             </div>
         </div>
 
+        <FormHeading class="text-red-700">Suspend Project</FormHeading>
+        <div class="flex flex-col lg:flex-row max-w-2xl space-y-4">
+            <div class="flex-grow">
+                <div class="max-w-sm pt-2">
+                    Once suspended, your project will not be available until restarted. Whilst suspend the project will consume no <span v-if="features.billing">billable</span> resources.
```
Comment (joepavitt) [inline frontend/src/pages/project/Settings/Danger.vue:85]:
```suggestion
                    Once suspended, your project will not be available until restarted. Whilst suspended, the project will consume no <span v-if="features.billing">billable</span> resources.
```

**Comment by joepavitt** [comment]
@hardillb Might be worth holding on this until https://github.com/flowforge/flowforge/pull/900 is merged, then we can utilise the new Dialog JS API, which streamlines the confirmation process here.

**Comment by hardillb** [comment]
Sounds like a plan

**Comment by joepavitt** [comment]
I've also just added some conditional disabling/messaging when a project is suspended.

Screenshot 2022-08-19 at 10 50 43

File: frontend/src/pages/project/Settings/Danger.vue [hunk 3]
```diff
@@ -185,6 +222,9 @@ export default {
             }).finally(() => {
                 this.loading.deleting = false
             })
+        },
+        suspendProject () {
+
```
Comment (hardillb) [inline frontend/src/pages/project/Settings/Danger.vue:227]: I assume we don't need this at all now?

Comment (joepavitt) [reply frontend/src/pages/project/Settings/Danger.vue:227]: good point- - I'll remove

**Review by hardillb — COMMENTED** [review]
I'd approve Joe's update, but github won't let me

**Review by joepavitt — APPROVED** [review]

**Pull request merged**